pkgs/by-name/lu/lunarvim/package.nix

```diff
@@ -0,0 +1,97 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
+, nodePackages
+, python311
```

Comment:
any reason for pining python to 3.11 ? 
3.11 is now default, was recently merged into staging https://github.com/NixOS/nixpkgs/pull/251878

Reply:
Because `pynvim` is only available in 310 and 311, so I picked the newer version.

Reply:
You could just use `python3` instead, which, as @kirillrdy said was recently updated to be "pointing" at python 3.11, and will also be pointing to up-to-date python versions in the future, making maintenance of lunarvim easier.